interfaces/cython/cantera/ck2cti.py

```diff
@@ -2132,7 +2132,8 @@ def writeCTI(self, header=None, name='gas', transportModel='Mix',
 
         return surface_names
 
-    def showHelp(self):
+    @classmethod
```

Comment:
I think this should be a `@staticmethod` since it doesn't use the `cls` parameter (or the old `self` parameter for that matter)